src/plugins/node-resolve.ts

```diff
@@ -14,8 +14,7 @@ export default (options: Options) => {
   const plugin = require('rollup-plugin-node-resolve')({
     extensions: ['.js', '.json', '.jsx', '.ts', '.tsx'],
     preferBuiltins: true,
-    jsnext: true,
-    module: true,
+    mainFields: ['module', 'jsnext:main', 'main', 'browser'],
```

Comment:
I guess `browser` shouldn't be the last one? It will never be used then

Reply:
Yes. it won't. What do you think to add it before main when `browser` is true?

Reply:
You did it :) Thanks :heart: